.github/workflows/ci.yml

```
@@ -30,6 +30,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+      - name: Install rdkafka
```

A little misleading since this step installs other stuff as well

```suggestion
      - name: Install native dependencies
```